extensions/adobe/experience/edge-autofilled-environment-details.schema.json

```diff
@@ -43,15 +43,53 @@
         }
       }
     },
-    "edge-environment-geo": {
+    "edge-environment-countrycode": {
       "properties": {
         "xdm:placeContext": {
           "type": "object",
           "properties": {
             "xdm:geo": {
               "title": "Geo",
-              "$ref": "https://ns.adobe.com/xdm/common/geo",
-              "description": "The geographic location where the experience was delivered based on the IP address from the client request."
+              "$ref": "https://ns.adobe.com/xdm/common/geo#/definitions/countrycode"
+            }
+          }
+        }
+      }
+    },
+    "edge-environment-postalcode": {
```

Comment:
I think here should be `stateProvince`, we will not be auto filling the postal code.

Reply:
Thanks for catching that. I've updated to use state province instead of postal code.